command/format/diff.go

```diff
@@ -1104,7 +1120,7 @@ func (p *blockBodyDiffPrinter) writeValueDiff(old, new cty.Value, indent int, pa
 					action = plans.Create
 				} else if new.HasIndex(kV).False() {
```

Comment:
I think this will panic if the map itself is marked.

Reply:
It does! This PR does not deal with non-primitive values.